### PR TITLE
Add tabulation site

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] add tabulation, fieldset and two columns checkbox in forms

https://github.com/open-orchestra/open-orchestra-model-bundle/pull/682
https://github.com/open-orchestra/open-orchestra-model-interface/pull/250
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1972
https://github.com/open-orchestra/open-orchestra-elastica-bundle/pull/59
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/317

### CODE REVIEW

**Comment by itkg-nanne** [comment]
update done @alavieille

File: Backoffice/Form/Type/SiteType.php [hunk 1]
```diff
@@ -65,46 +58,45 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
             ))
             ->add('blocks', 'oo_block_choice', array(
                 'multiple' => true,
-                'label' => 'open_orchestra_backoffice.form.website.blocks',
-                'required' => false
+                'expanded' => true,
+                'label' => false,
+                'required' => false,
+                'group_rank' => 3,
+                'sub_group' => 'open_orchestra_backoffice.form.website.sub_group.block',
```
Comment (ngilain) [inline Backoffice/Form/Type/SiteType.php]: @itkg-nanne : can you do the same process on sub_group that is done on group, ie introducing a sub_group_rank ?

File: Backoffice/Form/Type/Extension/FormTypeGroupExtension.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    /**
+     * @param FormView      $view
+     * @param FormInterface $form
+     * @param array         $options
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        if($form->getConfig()->getAttribute('group_enabled')) {
```
Comment (ngilain) [inline Backoffice/Form/Type/Extension/FormTypeGroupExtension.php]: @itkg-nanne : `if (`

**Comment by itkg-nanne** [comment]
update done @alavieille @ngilain

File: Backoffice/Tests/EventSubscriber/WebSiteNodeTemplateSubscriberTest.php [hunk 3]
```diff
@@ -50,7 +50,7 @@ public function setUp()
      *
      * @dataProvider provideSiteIdAndCountAddForm
      */
-    public function testPreSetData($siteId, $countAddForm)
+    public function testPreSetData($siteId, $disabled)
```
Comment (alavieille) [inline Backoffice/Tests/EventSubscriber/WebSiteNodeTemplateSubscriberTest.php]: phpdpc

File: Backoffice/Form/Type/Extension/FormTypeGroupExtension.php [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+
+/**
+ * Class FormTypeGroupExtension
+ */
+class FormTypeGroupExtension extends AbstractTypeExtension
```
Comment (alavieille) [inline Backoffice/Form/Type/Extension/FormTypeGroupExtension.php]: can you add unit test ?

**Pull request merged**